File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.java
class Solution {
    public long solution(int price, int money, int count) {
        long answer = 0l;
        long pay = 0l;
        for(int i = 1; i <= count; i++){
            pay += price * i;
        }
        if(pay > money){
            answer = pay - money;
        }

        return answer;
    }
}

File: 프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.swift
import Foundation

func solution(_ price:Int, _ money:Int, _ count:Int) -> Int64{
    var answer:Int64 = 0
    var pay:Int64 = 0
    for i in 1...count {
        pay += Int64(price * i)
    }
    if pay > money {
        answer = pay - Int64(money)
    }
    return answer
}